src/main/java/com/google/devtools/build/lib/skylarkbuildapi/repository/SkylarkRepositoryContextApi.java

```diff
@@ -294,6 +294,35 @@ public SkylarkExecutionResultApi execute(
   public boolean delete(Object path, Location location)
       throws EvalException, RepositoryFunctionExceptionT, InterruptedException;
 
+  @SkylarkCallable(
+      name = "patch",
+      doc =
+          "Apply a patch file to the root directory of external repository. "
+              + "The patch file should be a standard "
+              + "[unified diff format](https://en.wikipedia.org/wiki/Diff#Unified_format) file. "
+              + "The Bazel-native patch implementation doesn't support fuzz match and binary patch "
+              + "like the patch command line tool.",
+      useLocation = true,
+      parameters = {
+          @Param(
+              name = "patch_file",
+              allowedTypes = {
+                  @ParamType(type = String.class),
+                  @ParamType(type = Label.class),
+                  @ParamType(type = RepositoryPathApi.class)
+              },
+              doc =
+                  "Path of the patch file to apply."),
```

Comment:
Please document similar to https://github.com/bazelbuild/bazel/blob/dc0dccc8bbc3e676695fc114aa20f0e921bca2ba/src/main/java/com/google/devtools/build/lib/skylarkbuildapi/repository/SkylarkRepositoryContextApi.java#L60

(path can be a string, path, or label, and can be relative)

Reply:
Done